build.gradle

```diff
@@ -1,20 +1,71 @@
 buildscript {
-  ext.kotlin_version = '1.4.10'
+  ext.versions = [
+    'minSdk': 21,
+    'compileSdk': 30,
+    'targetSdk': 30,
```

Comment:
No real need to define these both. Libraries should only care about compileSdk and for their samples they can just propagate that as the target as well. The library should never propagate a target itself so the values will never deviate.